src/rgw/rgw_bucket.cc

```diff
@@ -648,16 +653,20 @@ int rgw_remove_bucket_bypass_gc(RGWRados *store, rgw_bucket& bucket,
   RGWRados::Bucket::List list_op(&target);
 
   list_op.params.list_versions = true;
+  list_op.params.allow_unordered = true;
```

Comment:
nice

Reply:
Surprising how useful this functionality is turning out to be....